fairseq/models/roberta/model.py

```diff
@@ -98,6 +98,8 @@ def add_args(parser):
                             help='scalar quantization noise and scalar quantization at training time')
         parser.add_argument('--untie-weights-roberta', action='store_true',
                             help='Untie weights between embeddings and classifiers in RoBERTa')
+        parser.add_argument('--do-r4f', action='store_true',
```

Comment:
Please rename this to something more descriptive, something like `--spectral-norm-classification-head`

Reply:
Updated.